docs/source/installation.rst

```diff
@@ -2,152 +2,199 @@ Installation
+Step 1: Setup dedicated python environment
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-    If you are using bash shell
-    ::
+    You can setup ``ivadomed`` using either Conda or Venv:
 
-        vim ~/.bashrc
+    .. tabs::
 
-    If you are using zsh shell
-    ::
-        
-        vim ~/.zshrc
+        .. tab:: Install via ``venv``
 
-Write in your .bashrc/.zshrc file with following line.
+            1. Setup Python Venv Virtual Environment.
 
-::
+                ``ivadomed`` requires Python >= 3.6 and <3.10.
 
-    export HDF5_USE_FILE_LOCKING='FALSE'
+                First, make sure that a compatible version of Python 3 is installed on your system by running:
 
-Save this change and restart your terminal to apply the change.
+                .. tabs::
 
-There are fundamentally two different approaches to install ``IvadoMed``:
+                    .. tab:: Mac/Linux
 
-1) Install via Conda
-    This is the easiest way for personal computers.
+                        .. code::
 
-2) Install via Venv/VirtualEnv
-    This is compatible with ComputeCanada cluster environment.
+                            python3 --version
 
-Approach 1: Conda
-------------------
+                    .. tab:: Windows
 
-Step 1: Clone the `ivadomed <https://github.com/ivadomed/ivadomed>`_ repository.
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+                        .. code::
 
-::
+                            python --version
 
-    git clone https://github.com/neuropoly/ivadomed.git
-    cd ivadomed
+                If your system's Python is not 3.6, 3.7, 3.8, or 3.9 (or if you don't have Python 3 installed at all), please `install Python <https://wiki.python.org/moin/BeginnersGuide/Download/>`_ before continuing.
 
-Step 2: Create new Conda Env called IvadoMedEnv (may take 5 to 15 minutes)
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+                Once you have a supported version of Python installed, run the following command:
 
-::
 
-    conda env create --file environment.yml
+                .. tabs::
 
-Step 3 : Activate environment and use
-++++++++++++++++++++++++++++++++++++++
+                    .. tab:: Mac/Linux
 
-::
+                        .. warning::
 
-    conda activate IvadoMedEnv
+                           If you use ``Debian`` or ``Ubuntu``, you may be prompted to install the ``python3-venv`` module when creating the virtual environment. This is expected, so please follow the instructions provided by Python. For other operating systems, ``venv`` will be installed by default.
 
-Step 4 : Install from source
-++++++++++++++++++++++++++++++++++++++
+                        Replacing ``3.X`` with the Python version number that you installed):
 
-::
+                        .. code::
 
-    pip install -e .
+                            python3.X -m venv ivadomed_env
```

Comment:
Nitpick: I'd maybe move some of the text around so that the command comes first:

```rst
                        .. code::

                           # Note: Replace ``3.X`` with the specific Python version number that you installed
                           python3.X -m venv ivadomed_env

                        .. warning::

                           If you use ``Debian`` or ``Ubuntu``, you may be prompted to install the ``python3-venv`` module when creating the virtual environment. This is expected, so please follow the instructions provided by Python. For other operating systems, ``venv`` will be installed by default.
```

(I suggest this because the preceding text is "run the following command:". I found it a bit jarring to see a warning, rather than the actual command. So, I think this way might flow a bit better?)

Reply:
I agree with Joshua here. I would also put the command first, followed by a note about Debian/Ubuntu.

Reply:
Great point.